internal/locale/locales/en-us.yaml

```diff
@@ -917,9 +919,9 @@ err_deprecation:
 
     Please update by running [ACTIONABLE]`state update`[/RESET].
 err_auth_required:
-  other: Authentication is required, please authenticate by running 'state auth'
+  other: Authentication is required. Please authenticate by running [ACTIONABLE]`state auth`[/RESET].
 err_auth_needinput:
-  other: You are logged out, in order to log in either run this command in interactive mode or provide your credentials via flags (see '[ACTIONABLE]state auth --help[/RESET]').
+  other: You are logged out. In order to log in, either run this command in interactive mode, or provide your credentials via flags (see '[ACTIONABLE]state auth --help[/RESET]').
```

Comment:
Please swap the single quotes to back tics and move them inside the actionable/reset tags.

Reply:
Okay. I changed other instances I found.